airflow/providers/docker/hooks/docker.py

```diff
@@ -37,6 +37,7 @@ class DockerHook(BaseHook, LoggingMixin):
     conn_name_attr = 'docker_conn_id'
     default_conn_name = 'docker_default'
     conn_type = 'docker'
+    hook_name = 'Docker Registry'
```

Comment:
This is not a docker registry, but a docker daemon

Reply:
Copied it form extras-packages.rst :D

Reply:
Actually, it is correct. This is a Docker Registry Hook not Daemon.

```
    Interact with a private Docker registry.

    :param docker_conn_id: ID of the Airflow connection where
        credentials and extra configuration are stored
    :type docker_conn_id: str
```

Reply:
Or not? I am looking deeper . Maybe it is mistake in documentation of the Hook :)

Reply:
Yeah. The PyDocs were wrong.

Reply:
It was all over the place "'Logging into Docker registry'", "No Docker registry URL provided" etc.

Reply:
Well. It can be used for both registry (with login) and Daemon :)

Reply:
I will leave it at "Docker"